pkg/helpers/retry_test.go

```diff
@@ -16,7 +16,7 @@ func TestRetryTimeout(t *testing.T) {
 	})
 	stop := time.Now().Sub(start)
 	if stop < 5*time.Second {
-		t.Errorf("retry ended too soon: %v", stop)
+		t.Errorf("retry ended too soon: %s", stop)
```

Comment:
Not that there's any issue with this change, but wondering if we want to just exclude `_test` files from the check?

Reply:
If we have a `%v` in a test that dumps, say, the API key that we use for testing, wouldn't that be an issue?  The whole PR is about being a bit paranoid about disclosures, so I'm happier checking the `_test` files too.